src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -191,10 +191,10 @@ void GcsPlacementGroupScheduler::PrepareResources(
         auto result = reply.success() ? Status::OK()
                                       : Status::IOError("Failed to reserve resource");
         if (result.ok()) {
-          RAY_LOG(DEBUG) << "Finished leasing resource from " << node_id
-                         << " for bundles: " << GetDebugStringForBundles(bundles);
+          RAY_LOG(INFO) << "Finished leasing resource from " << node_id
+                        << " for bundles: " << GetDebugStringForBundles(bundles);
         } else {
-          RAY_LOG(DEBUG) << "Failed to lease resource from " << node_id
+          RAY_LOG(ERROR) << "Failed to lease resource from " << node_id
```

Comment:
This shouldn't be an ERROR since this can happen normally due to conflicts?

Reply:
Right, I change to WARNING?